playbooks/common/openshift-cluster/openshift_hosted.yml

```diff
@@ -1,30 +1,33 @@
-- name: Create persistent volumes and create hosted services
+- name: Create persistent volumes
   hosts: oo_first_master
   vars:
-    attach_registry_volume: "{{ openshift.hosted.registry.storage.kind != None }}"
-    deploy_infra: "{{ openshift.master.infra_nodes | default([]) | length > 0 }}"
     persistent_volumes: "{{ hostvars[groups.oo_first_master.0] | oo_persistent_volumes(groups) }}"
     persistent_volume_claims: "{{ hostvars[groups.oo_first_master.0] | oo_persistent_volume_claims }}"
   roles:
   - role: openshift_persistent_volumes
     when: persistent_volumes | length > 0 or persistent_volume_claims | length > 0
-  - role: openshift_serviceaccounts
-    openshift_serviceaccounts_names:
-    - router
-    - registry
-    openshift_serviceaccounts_namespace: default
-    openshift_serviceaccounts_sccs:
-    - privileged
-  - role: openshift_registry
-    registry_volume_claim: "{{ openshift.hosted.registry.storage.volume.name }}-claim"
-    when: deploy_infra | bool and attach_registry_volume | bool
-  - role: openshift_metrics
-    when: openshift.hosted.metrics.deploy | bool
 
 - name: Create Hosted Resources
   hosts: oo_first_master
+  vars:
+    l_projects:
+      default:
+        default_node_selector: "{{ openshift_hosted_router_selector
+                                   | default(openshift_hosted_infra_selector
+                                   | default(none)) }}"
```

Comment:
@abutcher as we discussed, I think the sane default for the default project node selector would be '' rather than trying to determine what the value should be derived from the router/registry node selector. We should be passing in the router/registry node selector values into the router/registry creation commands anyway.